tests/integration/main.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 var verbose = flag.Bool("verbose", false, "turn verbose logging on/off")
-var multiconsumer = flag.Bool("multiconsumer", false, "run happy path and multiconsumer tests in parallel")
+var happyPathOnly = flag.Bool("happy-path-only", false, "run happy path tests only")
+var multiconsumer = flag.Bool("multiconsumer", false, "include multiconsumer tests in run")
```

Comment:
Flag naming consistency here could be nice 
```suggestion
var multi-consumer = flag.Bool("multiconsumer", false, "include multiconsumer tests in run")
```